github/pull_request.rb

```diff
@@ -55,11 +55,11 @@ def check_commits_style
       if (commit.commit.message.lines.first =~ /\A(fixes|refs) #\d+(, ?#\d+)*(:| -) .*\Z/i) != 0
         warnings += "  * #{commit.sha} must be in the format ```fixes #redmine_number - brief description```\n"
       end
-      if commit.commit.message.lines.first.size > 65
+      if commit.commit.message.lines.first && commit.commit.message.lines.first.chomp.size > 65
```

Comment:
I don't think the new checks for presence of the line are required, keep it as-is please.  (It'd fail earlier if this was a problem.)